src/main/java/com/icthh/xm/tmf/ms/resource/config/WebConfigurer.java

```diff
@@ -31,6 +25,10 @@
 import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
 import org.springframework.web.filter.CorsFilter;
 
+import javax.servlet.*;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version

+ wildcard imports